.github/workflows/release.yaml

```diff
@@ -15,21 +16,21 @@ jobs:
     with:
       push: true
       registry: registry.scality.com
-      namespace: ${{ github.event.repository.name }}
+      namespace: ${{ github.event.repository.name }}-dev
       name: ${{ github.event.repository.name }}
       context: .
       file: images/svc-base/Dockerfile
-      tag: ${{ github.event.inputs.tag }}-svc-base
+      tag: ${{ github.sha }}-svc-base
 
-  github-release:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Create Release
-        uses: softprops/action-gh-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          name: Release ${{ github.event.inputs.tag }}
-          tag_name: ${{ github.event.inputs.tag }}
-          generate_release_notes: true
-          target_commitish: ${{ github.sha }}
+#  github-release:
```

Comment:
Can we remove this instead of commenting out if it's not needed?

Reply:
The third commit will be dropped before merge, I needed a s3c image built to test the integration tests